src/main/java/seedu/taskell/model/task/TaskComplete.java

```diff
@@ -0,0 +1,38 @@
+package seedu.taskell.model.task;
+
+import java.util.Objects;
+
+public class TaskComplete {
+    
+    public static final String COMPLETED_INFO = "Completed";
+    public static final String NOT_COMPLETED_INFO = "Not Completed";
+    public final String isCompleted;
```

Comment:
Refactor isComplete to "taskStatus". Variable starting with is/has/can/should are reserved for boolean variables.
![image](https://cloud.githubusercontent.com/assets/13817632/19644542/e66b3d3a-9a22-11e6-99f6-cff3b0e51448.png)
![image](https://cloud.githubusercontent.com/assets/13817632/19644554/f87592f0-9a22-11e6-9654-4f0be77e5ed9.png)
![image](https://cloud.githubusercontent.com/assets/13817632/19644556/00de9234-9a23-11e6-8162-3ece852cffa9.png)